templates/README

```diff
@@ -1 +1,60 @@
-This contains some basic description about the BIDS dataset.
+### Introduction
+The README is usually the starting point for researchers using your data
+and serves as a guidepost for users of your data. A clear and informative
+README makes your data much more usable.
+
+Overview
```

Comment:
Maybe an easy low hanging fruit for the overview would be to copy-paste the summary printed out by the validator that says how many subjects are in there, what modality.